### PR TITLE
refactor: change border radius

### PR DESCRIPTION
**Pull request by leMaur**
## Summary
https://app.clickup.com/t/h96exw

## Checklist

-   [x] I checked my UI changes against the design and there are no notable differences
-   [x] I checked my UI changes for any responsiveness issues
-   [x] I checked my (code) changes for obvious issues, debug statements and commented code
-   [ ] I provided a screenshot of my changes to the component _(if applicable)_
-   [ ] I regenerated the `icons.html` file and checked if my newly added icon is shown correctly _(if necessary)_
-   [ ] I added an explanation on how to use the component to the readme _(if necessary)_
-   [ ] Documentation _(if necessary)_
-   [ ] Tests _(if necessary)_
-   [x] Ready to be merged

### CODE REVIEW

File: resources/assets/css/_document.css [hunk 1]
```diff
@@ -240,13 +240,13 @@ code.hljs .hljs-ln-n {
...
 .documentation-content .table-wrapper::-webkit-scrollbar {
-    @apply rounded-lg bg-theme-primary-100;
+    @apply rounded-xl bg-theme-primary-100;
     width: 6px;
     height: 6px;
 }
 
 .documentation-content .table-wrapper::-webkit-scrollbar-thumb {
-    @apply rounded-lg bg-theme-primary-200;
+    @apply rounded-xl bg-theme-primary-200;
```
Comment (ItsANameToo) [inline resources/assets/css/_document.css:249]: same scrollbar question

File: resources/assets/css/_components.css [hunk 2]
```diff
@@ -159,15 +159,15 @@
 }
 
 .pagination-pages-mobile {
-    @apply flex justify-center px-2 mb-3 rounded bg-theme-primary-100;
+    @apply flex justify-center px-2 mb-3 rounded-xl bg-theme-primary-100;
```
Comment (ItsANameToo) [inline resources/assets/css/_components.css:162]: this is technically a button, so should be `rounded`. it affects the top button on mobile
image

File: resources/assets/css/_components.css [hunk 3]
```diff
@@ -159,15 +159,15 @@
...
 }
 
 .pagination-button-mobile {
     @apply flex justify-center flex-1;
 }
 
 .pagination-pages {
-    @apply hidden px-2 rounded bg-theme-primary-100;
+    @apply hidden px-2 rounded-xl bg-theme-primary-100;
```
Comment (ItsANameToo) [inline resources/assets/css/_components.css:170]:
```suggestion
    @apply hidden px-2 rounded bg-theme-primary-100;
```

**Review by ItsANameToo — CHANGES_REQUESTED** [review]

File: resources/assets/css/_components.css [hunk 4]
```diff
@@ -1,6 +1,6 @@
 /** Dropdowns **/
 .dropdown-button-outline {
-    @apply bg-transparent border border-solid rounded border-theme-secondary-200;
+    @apply bg-transparent border border-solid rounded-xl border-theme-secondary-200;
```
[the remark this answers is not in the document]
Comment (leMaur) [reply resources/assets/css/_components.css:3]: @ItsANameToo It's used in Hermes https://github.com/ArkEcosystem/laravel-hermes/blob/10acc56cd4b2b6bddd545febcc04229720014d89/resources/views/components/manage-notifications.blade.php#L20

[the remark this answers is not in the document]
Comment (ItsANameToo) [reply resources/assets/css/_components.css:3]: ah right, then it should stay `rounded` as it's part of the button

File: resources/assets/css/_components.css [hunk 5]
```diff
@@ -27,7 +27,7 @@
 
 /** Dropdown Scrollbar **/
 .dropdown-scrolling::-webkit-scrollbar {
-    @apply rounded-lg;
+    @apply rounded-xl;
```
[the remark this answers is not in the document]
Comment (leMaur) [reply resources/assets/css/_components.css:30]: @ItsANameToo border radius on ::-webkit-scrollbar seems ignored (Chrome and Safari).

File: resources/assets/css/_components.css (hunk 4 above)
[the remark this answers is not in the document]
Comment (leMaur) [reply resources/assets/css/_components.css]: @ItsANameToo yep! Done 👍🏽

**Review by ItsANameToo — APPROVED** [review]

**Pull request merged**